File: src/main/java/xyz/onegrid/coreapi/repository/AuthorityRepository.java
package xyz.onegrid.coreapi.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import xyz.onegrid.coreapi.domain.Authority;

/**
 * Spring Data JPA repository for the {@link Authority} entity.
 */
public interface AuthorityRepository extends JpaRepository<Authority, String> {}

File: src/main/kotlin/xyz/onegrid/coreapi/repository/AuthorityRepository.kt
package xyz.onegrid.coreapi.repository

import org.springframework.data.jpa.repository.JpaRepository
import xyz.onegrid.coreapi.domain.Authority

/**
 * Spring Data JPA repository for the [Authority] entity.
 */

interface AuthorityRepository : JpaRepository<Authority, String>
